README.md

```diff
@@ -3,7 +3,7 @@
 
 [![npm](https://img.shields.io/npm/v/@finos/perspective.svg?style=flat)](https://www.npmjs.com/package/@finos/perspective)
 [![PyPI](https://img.shields.io/pypi/v/perspective-python.svg?style=flat)](https://pypi.python.org/pypi/perspective-python)
-[![Build Status](https://dev.azure.com/finosfoundation/perspective/_apis/build/status/finos.perspective?branchName=master)](https://dev.azure.com/finosfoundation/perspective/_build/latest?definitionId=1&branchName=master)
+[![Build Status](https://github.com/finos/perspective/workflows/Build%20Status/badge.svg?branch=master)](https://github.com/finos/perspective/actions?query=workflow%3A%22Build+Status%22)
```

Comment:
GHA may not become the default in this PR - let's split this (and anything else so marked below) in a "move GHA to prod" PR